packages/create-bison-app/template/tests/utils.tsx

```diff
@@ -17,58 +23,29 @@ export { default as userEvent } from '@testing-library/user-event';
  */
 export function render(ui: RenderUI, { router = {}, ...options }: RenderOptions = {}) {
   return defaultRender(ui, {
-    wrapper: ({ children }) => {
+    wrapper: function Wrapper({ children }) {
+      const [queryClient] = useState(() => new QueryClient());
+
+      const [trpcClient] = useState(() =>
+        trpc.createClient({
```

Comment:
One thing this doesn't do is provide a way to mock requests. We could add that feature here - a special link which takes inputs and returns some static output provided in the body of the test.

Reply:
I think that's a good thing, as it discourages mocking until we absolutely need it.

Reply:
What would be really cool is if we had bulletproof Stripe/Twilio mocks and SQLite support in Bison. Then, tests run e2e without needing a test database running, and only our third-party APIs are mocked.

Reply:
We could set up tRPC to use a special client-side link (instead of the BatchHTTPLink) that calls the tRPC procedures directly instead of over HTTP